Gemfile

```diff
@@ -23,10 +23,16 @@ group :development do
 end
 
 # Required to make imports in Active Admin stylesheet work
-gem 'sassc-rails', '~> 1.0'
+gem 'sassc-rails', '~> 2.1'
 
 # Required for XML serialization in Active Admin
 gem 'activemodel-serializers-xml'
 
+# Make webpacker available in specs. Host applications that want to
+# use webpacker need to add it to their Gemfile themselves. Requiring
+# webpacker in an engine file (like we normally do) would force all
+# host application to install webpacker.
+gem 'webpacker'
+
```

Comment:
This is now on master. The commit on this branch should be gone after the next rebase.